New package: MaximumIndependentSet v0.1.1

JuliaRegistrator opened the pull request:
- Registering package: MaximumIndependentSet
- Repository: https://github.com/claud10cv/MaximumIndependentSet.jl
- Created by: @claud10cv
- Version: v0.1.1
- Commit: 00881891dd28907524f7fed4ea1ed6bb80a9cd6d
- Reviewed by: @claud10cv
- Reference: https://github.com/claud10cv/MaximumIndependentSet.jl/commit/00881891dd28907524f7fed4ea1ed6bb80a9cd6d#commitcomment-140169753
- Description: Julia wrapper for the maximum independent set method
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6c10e14e525da41189dbb6c5319c42902940b56bebebdacc1bbc556b7263af6ced4e42e31d8efcee34606be11ebaa859cb34bc837cd279105bc08f1efa35c73b5e72afad577d05758143af54d08d219bcaf333fc3d6a14dd6fdfce92536c1ab9fe55d736466156f33454e048a3de0155c837adf06dfbb4c2325aacac82bdb79e7702be3e1f95514b33d1acef48d9206435e3094f36e431d11ef254d67ae3de881b61f9f870763a5006a75fab0428829db56786551ef2d06de45bef96240c4efc5 -->

gdalle commented:
Can I again offer you to choose a more specific name for the package since it is one single wrapper?

claud10cv commented:
I updated the package and implemented a MIP solver + a heuristic for MIS. Do you still believe it would be a good idea to change the name?

gdalle commented:
[noblock]
In that case indeed it makes sense to keep that name, thanks 😊
If you want the registration to succeed, put [noblock] in your comments and fix the issues raised by the bot

JuliaRegistrator committed:
New package: MaximumIndependentSet v0.1.1

UUID: 9e812ceb-42a3-473a-b295-b8bb2099a8ce
Repo: https://github.com/claud10cv/MaximumIndependentSet.jl.git
Tree: a9a0b2059dd416f2ab8863e12449ed30d272fd57

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
